src/renderer.cpp

```diff
@@ -690,6 +701,16 @@ void Renderer::fontBlur(f32 s)
     pushCmd(cmd);
 }
 
+void Renderer::fontFace(char* s)
+{
+	nvgFontFace(mNanoVg, s); // Must call nanovg because affects text size query
+
+	DrawCmd cmd;
+	cmd.type = DrawCmdType_fontFace;
+	strcpy(cmd.s.str, s);
```

Comment:
probably want to use strncpy like the text() function, if the length of s is greater than sizeof(str) it will write out of bounds